dockerfiles/analytics/Dockerfile

```diff
@@ -29,13 +30,19 @@ ARG USER_HOME=/home/${USER}
 # set dependant files directory
 ARG FILES=./files
 # set jdk configurations
-ARG JDK_DIST=jdk1.8.0*
 ARG JAVA_HOME=${USER_HOME}/java
+ARG JDK_NAME=jdk-8u251-linux-x64.tar.gz
+ARG JDK_URL
 # set wso2 product configurations
 ARG WSO2_SERVER=wso2ei
 ARG WSO2_SERVER_VERSION=6.3.0
 ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER_PACK}
+ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/product-ei/releases/download/v6.3.0/wso2ei-6.3.0.zip
```

Comment:
@NishikaDeSilva shall we build the URL using the build arguments which we have set earlier?

For example,

```
ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/product-ei/releases/download/v${WSO2_SERVER_VERSION}/${WSO2_SERVER_PACK}.zip
```